src/components/pages/pages.spec.js

```diff
@@ -0,0 +1,268 @@
+import React from 'react';
+import CSSTransitionGroup from 'react-transition-group/CSSTransitionGroup';
+import { shallow, mount } from 'enzyme';
+import BasePages, { Page } from './pages.component';
+import Heading from '../heading/heading';
+import { rootTagTest } from '../../utils/helpers/tags/tags-specs/tags-specs';
+import {
+  PagesContent
+} from './pages.style';
+import StyledHeadingIcon from '../heading/heading.style';
+import classicTheme from '../../style/themes/classic';
+import { assertStyleMatch } from '../../__spec_helper__/test-utils';
+import 'jest-styled-components';
+
+describe('BasePages', () => {
+  let wrapper, page;
+
+  beforeEach(() => {
+    wrapper = shallow(
+      <BasePages
+        theme={ classicTheme }
+        className='foobar'
+        initialPageIndex={ 0 }
+      >
+        <Page title='Page One' />
+        <Page title='Page Two' />
+        <Page title='Page Three' />
+      </BasePages>
+    );
+
+    page = wrapper.instance().visiblePage();
+  });
+
+  describe('componentWillMount', () => {
```

Comment:
Does this lifecycle method even exist in pages?

Reply:
good point, it does not therefore I am removing related specs as they are obsolete.